requirements.txt

```diff
@@ -1,28 +1,12 @@
-wheel==0.30.0
 wxPython==4.0.1
-configobj==5.0.6
 docker==2.6.1
-json-tricks==3.8.0
-olefile==0.44
-PyOpenGL==3.1.0
-pyOpenSSL==17.5.0
+construct==2.8.14
 mne==0.17.0
-lxml==4.1.1
-PsychoPy==3.0.4
-pytz==2017.2
-six==1.11.0
-numpy==1.16.2
-scipy==1.2.1
+PsychoPy==2020.2.4.post1
+numpy==1.19.2
+scipy==1.5.2
 sklearn==0.0
 seaborn==0.9.0
-construct==2.8.14
 matplotlib==2.1.1
 pylsl==1.13.1
-psutil==5.4.0
-pandas==0.24.1
-sounddevice==0.3.10
-SoundFile==0.10.1
-PySoundCard==0.5.2
-PySoundFile==0.9.0
-Pillow==4.3.0
-openpyxl==2.6.3
```

Comment:
We use some of these dependencies directly (ex. sounddevice and psutil). I'm not sure what best practices are but if I'm reading the docs correctly it seems like we should retain them either here or in install_requires (setup.py) even though they are included transitively from psychopy.

Reply:
Good callout. I added back all the requirements I could find that we explicitly use. There are a few cases we might refactor to use PsychoPy methods now. When I first implemented sound feedback, PsychoPy sound wasn't working. It seems more stable now.